internal/proto/hotstuffpb/convert.go

```diff
@@ -1,14 +1,14 @@
-// Package hotstuffpb contains protocol buffers message types and conversion functions for the HotStuff protocol.
+// Package convert contains conversion methods between protocol buffer message types and HotStuff protocol message structures.
```

Comment:
This is now back in the `hotstuffpb` package. Also, I think they are functions, not methods.